clientv3/client.go

```diff
@@ -77,6 +77,11 @@ func New(cfg Config) (*Client, error) {
 	return newClient(&cfg)
 }
 
+func NewCtxClient(ctx context.Context) *Client {
```

Comment:
Add a comment on this Client? Probably document that this client is defined for internal usage?